src/index.ts

```diff
@@ -8,6 +8,6 @@ export * as francium from "./francium";
 export * as orca from "./orca";
 export * as tulip from "./tulip";
 export * as nftFinance from "./nftFinance";
-export * as utils from "./utils";
```

Comment:
Tbh this might be a better approach since it gives all util functions a shared domain. What's your take?

Reply:
Agree, then I'll change it back and use `utils.getMultipleAccounts` in gateway.